apps/sms/index.html

```diff
@@ -125,6 +125,7 @@
     <script defer src="services/js/activity/activity_client.js"></script>
     <script defer src="services/js/messaging/messaging_client.js"></script>
     <script defer src="services/js/moz_mobile_connections/moz_mobile_connections_client.js"></script>
+    <script defer src="services/js/moz_settings/moz_settings_client.js"></script>
```

Comment:
__issue:__ Could you please also take care about split view, New Message view in split mode is broken with the PR (actually if you rebase your PR on master it should fail just revived split inbox integration test, if not then my test is bad :/, that's why I want to return all our integration tests as soon as possible :D)